driver/src/r8126.h

```diff
@@ -722,7 +722,9 @@ This is free software, and you are welcome to redistribute it under certain cond
 
 #define rtl8126_rx_page_size(order) (PAGE_SIZE << order)
 
-#define MAX_NUM_TX_DESC 1024    /* Maximum number of Tx descriptor registers */
+// KASTEN - Limit size due to large continuous DMA block needed for performance.
+//          TODO: This limitation can most likely be lifted after some code changes.
+#define MAX_NUM_TX_DESC 256     /* Maximum number of Tx descriptor registers */
```

Comment:
This tanks tx performance for more than one thread is sending. We are talking about going down to 10kpps, where single threaded it does ~500kpps

Reply:
Addressed in commit 2b055eb75735f4a2a8ab8e5221080a5c9716b129